packages/components/scripts/pngToDataurl.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * AUTO-GENERATED FILE CREATOR
+ *
+ * This script reads all `.png` files inside `pngFixtures/`,
+ * converts them into base64 data URLs, and writes them into
+ * `imageFixtures.ts` for use in Storybook + Playwright tests.
+ *
+ * Run with: `yarn generate:image-fixtures`
+ */
+const fs = require('fs');
+const path = require('path');
+
+const inputFolder = path.posix.resolve(__dirname, '../config/playwright/setup/pngFixtures');
+const outputFile = path.posix.resolve(__dirname, '../config/playwright/setup/utils/imageFixtures.ts');
+
+// Load existing fixtures if any
+let existingFixtures = {};
+if (fs.existsSync(outputFile)) {
+  try {
+    const content = fs.readFileSync(outputFile, 'utf8');
+    const match = content.match(/export const imageFixtures = (.*) as const;/s);
+    if (match) {
+      existingFixtures = JSON.parse(match[1]);
+    }
+  } catch (e) {
+    console.warn('⚠️ Could not parse existing imageFixtures.ts, regenerating...');
+  }
+}
+
+// Get PNG files
+const files = fs.readdirSync(inputFolder).filter(f => f.endsWith('.png'));
+const newFixtures = { ...existingFixtures };
+let changed = false;
+
+files.forEach(file => {
+  const filePath = path.posix.join(inputFolder, file);
+  const buffer = fs.readFileSync(filePath);
+  const base64 = buffer.toString('base64');
+  const key = file.replace(/\.png$/, '');
+  const dataUrl = `data:image/png;base64,${base64}`;
+
+  if (newFixtures[key] !== dataUrl) {
+    newFixtures[key] = dataUrl;
+    changed = true;
+  }
+});
+
+if (changed) {
+  const output = `/**
+ * AUTO-GENERATED FILE – DO NOT EDIT
+ * Run \`yarn generate:image-fixtures\` to regenerate.
+ */
+
+export const imageFixtures = ${JSON.stringify(newFixtures, null, 2)} as const;
```

Comment:
question: Is it really necessary to pass the 'space' value?
```suggestion
export const imageFixtures = ${JSON.stringify(newFixtures)} as const;
```

Reply:
It’s not necessary for the script to work, but it’s useful for maintainability. I think, we should keep it.